index.js

```diff
@@ -1,81 +1,125 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+let counter = 1
+counter = 2
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
 
-// console.log(name)
+const name = "Raffy"
+
+console.log("%cQuestion 2", "color: red") 
+
+console.log(name)
 // => "Raffy"
 // name = "Not Raffy" 
 // => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
+function drinkWater (thirst_level) {
+    console.log("Man I sure am thirsty")
+    console.log("Ahh that hits the spot")
+    console.log(thirst_level - 1)
+}
+
```

Comment:
Almost! Just a few notes:

- In Javascript, it's standard to use camel case for variable and functions names. So instead of `thirst_level`, the variable would be `thirstLevel`

- In order to actually change the value of `thirst_level`, you have to do this:

```javascript
thirstLevel = thirstLevel - 1
```
- Your function should explicitly return the updated thirst_level (there are no implicit returns in Javascript), so after line 36, you should have:
```
return thirstLevel
```